### ISSUE TITLE
Arbitray image pair in flownet

### ISSUE THREAD

**Issue by onlytailei** [ClementPinard/FlowNetPytorch#5]
Based on flownet paper:

> We do not have any fully connected layers, which allows the networks to take images of arbitrary size as input.

But it seems that if I input a different size image pair (not 384*512). The network will give an error.

**Comment by ClementPinard** [comment]
Your image have to be a 64x64 multiple in size.

If it' not the case, you can make a black canvas that fulfill this specification, and put your image inside of it, and assume null flow outside of your image

Say you want the flowmap of a 120x120 image, just make a black 128x128 image and put your 120x120 image at the center, that way it will work.

Good luck :

**Comment by onlytailei** [comment]
Hi @ClementPinard , thanks for you reply.
Another problem about the preprocessing, cause the original paper do not say it clearly.
If I have a image pair, and I hope to get the optical flow of this pair.
What I should do is to transpose them to BGR, normalize them to [0,1], and use the FlownetS [network](https://github.com/ClementPinard/FlowNetPytorch/blob/master/models/FlowNetS.py#L32) and load the weight from [here](https://drive.google.com/drive/folders/0B5EC7HMbyk3CbjFPb0RuODI3NmM) (I find a pretrained pytorch pth file here).  Right?

**Comment by ClementPinard** [comment]
Yes, the pretrained model differ from the pytorch models, because it does not take the same input, especially the range which is [0,1] instead of mean,std = [0,1]
However, BGR is not that important. In fact, data augmentation during flownet training included color warping, so the model is supposed to handle color inversion such as BGR -> RGB, so it should stay decent on RGB images.
